lib/bolt/applicator.rb

```diff
@@ -284,6 +284,7 @@ def apply_ast(raw_ast, targets, options, plan_vars = {})
                   'catalog' => Puppet::Pops::Types::PSensitiveType::Sensitive.new(catalog),
                   'plugins' => Puppet::Pops::Types::PSensitiveType::Sensitive.new(plugins),
                   'apply_settings' => @apply_settings,
+                  'bolt_builtin_content' => @modulepath.full_modulepath - @modulepath.user_modulepath || [],
```

Comment:
Instead of doing this calculation, we should add a new function to https://github.com/puppetlabs/bolt/blob/main/lib/bolt/config/modulepath.rb and document what it is there

Reply:
YEah, i will have to do a significant amount of cleanup i think around all the mocking etc in tests. Just wanted to show this approach (and test it).

Reply:
Ah, I see.

Yeah this approach is a good way to go

Reply:
Actually, it looks like we just pass in resolved arrays into applicator anyway. I have tested this change on a fresh vmpooler image and it appears to work. Looks like our tests dont blow up with an extra task parameter so i'm just going to leave the comment and call it good.